### ISSUE TITLE
get only validate data

### ISSUE THREAD

**Issue by imanborumand** [urameshibr/lumen-form-request#6]
hello 
how can get only validated data?

**Comment by zubairIftikharKwe** [comment]
@imanborumand  Any Solution to get Validated data?

**Comment by deusmar** [comment]
@imanborumand @zubairIftikharKwe => https://github.com/urameshibr/lumen-form-request/pull/10

**Comment by imanborumand** [comment]
> @imanborumand Any Solution to get Validated data?

yes i use custom form request and add one method by name getData
in this method i add items and get validate in controller
